app/templates/buy-quick-start.pug

```diff
@@ -70,7 +68,9 @@
               popup-name="coinify-verification"
               translate="WHY")
         .mts.pos-rel(ng-show="tradingDisabled && tradingDisabledReason === 'awaiting_first_trade_completion'")
-          span.pos-abs.type-sm(ng-show="isPendingTradeState('awaiting_transfer_in')")
+          span.pos-abs.type-sm(ng-show="isPendingSellTrade")
+            span.state-danger-text(translate="SELL.WAIT_FOR_TRADE_COMPLETE")
+          span.pos-abs.type-sm(ng-show="isPendingTradeState('awaiting_transfer_in')" ng-hide="isPendingSellTrade")
```

Comment:
Bad practice to use `ng-show` and `ng-hide` on the same element (can lead to weird display issues). Could you pick one and use boolean logic to decide whether to show / hide?